include/eve/detail/function/simd/arm/sve/load.hpp

```diff
@@ -22,7 +22,7 @@ template<relative_conditional_expr C,
 EVE_FORCEINLINE wide<T, N>
 load_(EVE_SUPPORTS(sve_),
   C const& cond, safe_type const& s, eve::as<wide<T, N>> const& tgt, Ptr p)
-  requires (!has_bundle_abi_v<wide<T, N>>)
+  requires (sve_abi<abi_t<T, N>>)
```

Comment:
was n't the old test requried ? Is it what we do in other arch ?

Reply:
sve_abi includes not bundle